distribution/openrct2.d.ts

```diff
@@ -2538,7 +2538,7 @@ declare global {
         ButtonWidget | CheckboxWidget | ColourPickerWidget | CustomWidget | DropdownWidget | GroupBoxWidget |
         LabelWidget | ListViewWidget | SpinnerWidget | TextBoxWidget | ViewportWidget;
         
-    type IconName = "empty" | "placeholder" | "logo" | "logo_text" | "fast_forward" | "game_speed_indicator" |
+    enum IconName = "empty" | "placeholder" | "logo" | "logo_text" | "fast_forward" | "game_speed_indicator" |
```

Comment:
This isn't valid a valid Typescript enum, the `type` was correct. @ltsSmitty just asked for the strings to be alphabetical order. :)